dc/core/resources/BorderRectangle.qml

```diff
@@ -39,7 +39,7 @@ Item {
         width: Style.borderWidth
         height: Style.borderWidth
         radius: Style.borderWidth
-        color: Style.inactiveBorderColor
+        color: "#00000000"
```

Comment:
Ah, but making it invisible can be controlled with the visible or opacity property. The color should not be used imo.